### DISCUSSION TITLE
Logo problem when running. It's bigger.

### DISCUSSION THREAD

**Discussion by HaiDevX** [elementary/triage#660] [Q&A]
### The logo problem appears at startup as shown in the picture.
The system works very well, but I think that this problem may be bad for some people, so I shared it with you so that it can be fixed and contribute to this wonderful distribution.

![IMG_20241220_043945](https://github.com/user-attachments/assets/d2bc9292-2eb7-4308-9247-31a88cbb1351)

- By the way, it works on my computer, which is from 2010.😳😁
- 128g ssd
- 8g ram
- Intel core i3 First generation.

- This is the latest version elementary OS 8.

**Greetings** to you// Haitham

**Comment by itsyumiki** [comment]
I have the same problem, it annoys me...

**Comment by ryonakano** [comment]
Closing as duplicate of https://github.com/elementary/plymouth-theme/issues/31
